AzurePSDriveStorageAccount.psm1

```diff
@@ -298,3 +419,42 @@ class Blob : SHiPSDirectory
     }
 }
 
+
+Function SetContentUtility()
+{
+    param(
+        [string]$Content,
+        [string]$Path,
+        [string]$ShareName,
+        [string]$FolderName,
+        [object]$Context
+    )
+
+    # Save the text content to a local temp file because the Set-AzureStorageFileContent cmdlet takes file only
+    $tmpfile = [System.IO.Path]::GetTempFileName()
+
+    try {
+
+        Microsoft.PowerShell.Management\Set-Content -Path $tmpfile -Value $Content
+
+        # $content is the 'Value' passed in from Set-Content
+        # $path is the full path. e.g., Azure:\AutomationTeam\StorageAccounts\myaccount\Files\HelloWorld\hello.ps1".
```

Comment:
Remove references to internal subscription or resource names

Reply:
Updated.